src/Chapter3.hs

```diff
@@ -970,7 +1061,25 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
-
+data Weekday = MON | TUE | WED | THU | FRI | SAT | SUN deriving (Enum, Show, Eq)
+instance Bounded Weekday where
+  minBound = MON
+  maxBound = SUN
```

Comment:
The `Bounded` typeclass is one of the standard typeclasses and it can be derived as well :slightly_smiling_face: Moreover, it would be preferable to derive it, because if you change the data type definition (to the "American" system, for instance), you can forget to update the manual instance, but the derived instance always remains up-to-date.